test/integrations/validation/_fixtures/Validator.fixtures.ts

```diff
@@ -90,6 +91,35 @@ const ARRAY_VALIDATION: ValidationSchema = {
     }
 };
 
+const HTTP_HTTPS_PROTOCOLS_URL_VALIDATION: ValidationSchema = {
+    url: {
+        message: INVALID_URL,
+        URL: {
+            required: true,
+            protocols: ['http', 'https']
+        }
+    }
+};
+
+const NO_PROTOCOL_URL_VALIDATION: ValidationSchema = {
+    url: {
+        message: INVALID_URL,
+        URL: {
+            required: true
+        }
+    }
+};
+
+const HTTPS_FTP_URL_VALIDATION: ValidationSchema = {
```

Comment:
This scheme doesn't seem to be used. As discussed I prefer using this scheme over the HTTP_HTTPS version because it has two clearly distinct protocols. With this scheme we can also test if the HTTP protocol gets rejected when the HTTPS protocol is configured. Meaning that validation is smart enough to look beyond the startsWith when it comes to protocol validation.